_includes/articles/cardcoins-rbf-batching.md

```diff
@@ -0,0 +1,48 @@
+{:.post-meta}
+*by [CardCoins][]*
+
```

Comment:
If we want to include an short definition at the opening of the blog post, here is a suggestion: 

"Additive batching" is a scheme whereby additional outputs are included to unconfirmed transactions in the mempool. This field report outlines efforts CardCoins has in implementing a reorg and DoS safe implementation of such a scheme.

Will let you guys ultimately decide on its inclusion. I could go either way.